Unintentional breaking API change in 0.1.18 from 0.1.17

After running `cargo update` for a crate that I'm working on, I encountered a build failure due to the changes included in 0.1.18. This issue does not exist in 0.1.17. Patch updates should not be breaking API changes according to semver.

```
error[E0277]: the trait bound `lru::KeyRef<uuid::Uuid>: std::borrow::Borrow<&uuid::Uuid>` is not satisfied

1220 |         match cache.get_mut(&id) {
     |                     ^^^^^^^ the trait `std::borrow::Borrow<&uuid::Uuid>` is not implemented for `lru::KeyRef<uuid::Uuid>`
     |
     = help: the following implementations were found:
               <lru::KeyRef<K> as std::borrow::Borrow<K>>

error: aborting due to previous error
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## [v0.2.0](https://github.com/jeromefroe/lru-rs/tree/0.2.0) - 2019-09-27
+
+- Release a new minor version because of an accidental breaking change in the previous release ([#50](https://github.com/jeromefroe/lru-rs/issues/50)).
+
 ## [v0.1.18](https://github.com/jeromefroe/lru-rs/tree/0.1.18) - 2019-09-25
 
 - Add borrowed type support for get_mut.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "lru"
-version = "0.1.18"
+version = "0.2.0"
 authors = ["Jerome Froelich <jeromefroelic@hotmail.com>"]
 description = "A LRU cache implementation"
 homepage = "https://github.com/jeromefroe/lru-rs"

